docs/misc/markdown.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 5 minutes Markdown guide
+date: 2017-12-21
+author: [Dima German]
+---
+
+Markdown is an easy-to-use markup language, used to format docs for web, using plain text.  
+Used by datahub, github, bitbucket and many many more.
+
+### text:
```

Comment:
I think this should be capitalized and we don't need `:`

Reply:
fixed